bin/svn-git-up

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Update Git and SVN repos checked out in the same directory.
+#
+# Pass a branch name like 4.6 as the sole argument to checkout that branch. Otherwise, master/trunk is checked out.
+
+cd $(dirname "$0")/../public/core-dev
```

Comment:
Can we remove this?

I think we should be flexible to run this script from whichever directory we like, it shouldn't be tied to this project's structure. Furthermore, the correct location is already taken care of via the `git` alias.

Reply:
How about changing it as follows:

```suggestion
cd $( git rev-parse --show-toplevel )
```

This would eliminate needing to run it as part of the `alias`, and it would ensure the script runs from the Git repo root of whatever the current working directory is.

Reply:
SGTM!